### PR TITLE
Implement functionality for supplier commands and test cases

### PR DESCRIPTION
**Pull request by weesiau**

### CODE REVIEW

**Comment by Bahamas20** [comment]
All LGTM !

**Review by Bahamas20 — APPROVED** [review]
LGTM !

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 1]
```diff
@@ -76,8 +77,12 @@ public void equals() {
         editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
 
-        // different email -> returns false
-        editedAlice = new PersonBuilder(ALICE).withEmail(VALID_EMAIL_BOB).build();
+        // different price -> returns false
```
Comment (hauchongtang) [inline src/test/java/seedu/address/model/person/PersonTest.java:80]: What if there are Suppliers with similar products with the same price ?

**Review by hauchongtang — APPROVED** [review]
LGTM, for the comment I gave, we can de-conflict this later.

**Comment by dreammac3816547290** [comment]
Looks good!

**Review by dreammac3816547290 — APPROVED** [review]
Looks good, can merge.

**Review by Kang-Quan — APPROVED** [review]
LGTM!

**Pull request merged**